go.mod

```diff
@@ -4,9 +4,9 @@ go 1.19
 
 require (
 	github.com/hashicorp/golang-lru v0.5.4
-	github.com/hyperledger/firefly-common v1.2.10
-	github.com/hyperledger/firefly-signer v1.1.7
-	github.com/hyperledger/firefly-transaction-manager v1.2.11
+	github.com/hyperledger/firefly-common v1.2.11
```

Comment:
noting I think this now supersedes https://github.com/hyperledger/firefly-evmconnect/pull/83